scripts/build-dist.js

```diff
@@ -0,0 +1,92 @@
+import * as p from 'path';
+import * as fs from 'fs';
+import rollup from 'rollup';
+import babel from 'rollup-plugin-babel';
+import npm from 'rollup-plugin-npm';
+import commonjs from 'rollup-plugin-commonjs';
+import replace from 'rollup-plugin-replace';
+import UglifyJS from 'uglify-js';
+
+const copyright = (
+`/*
+ * Copyright ${new Date().getFullYear()}, Yahoo Inc.
+ * Copyrights licensed under the New BSD License.
+ * See the accompanying LICENSE file for terms.
+ */
+`
+);
+
+function createBundle() {
+    return rollup.rollup({
+        entry: p.resolve('src/react-intl.js'),
+        plugins: [
+            babel({
+                presets: [
+                    'es2015-rollup',
+                    'react',
+                ],
+            }),
+            npm({
+                jsnext: true,
+                skip: [
+                    'react',
+                ],
+            }),
+            commonjs({
+                sourceMap: true,
+            }),
+            replace({
+                'process.env.NODE_ENV': JSON.stringify(process.env.NODE_ENV),
```

Comment:
There's a dev build. The default value is `undefined`, I can explicitly set it to `"development"` in the `package.json` `scripts`.